model_compression_toolkit/exporter/model_wrapper/keras/validate_layer.py

```diff
@@ -39,40 +38,34 @@ def is_keras_layer_exportable(layer: Any) -> bool:
         if isinstance(layer, InputLayer):
             return True
 
-        valid_layer = isinstance(layer, KerasQuantizationWrapper)
+        valid_layer = isinstance(layer, Layer)
```

Comment:
Same. I don't understand this change.

Reply:
As part of the exporter's validation of a model, it should ensure all layers are of this type. For example, it does not support layers that are sequential models. For this reason, it validates it.